### PR TITLE
[RFR] [FrontController] #221 application's dispatcher will dispatch event before and after request process

### PR DESCRIPTION
**Pull request by eric-chau**
See #221 for more details.

### CODE REVIEW

File: FrontController/Event/PostResponseEvent.php [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+     */
+    private $request;
+
+    /**
+     * Create a new instance of PostResponseEvent.
+     *
+     * @param Response $response
+     * @param Request  $request
+     */
+    public function __construct(Response $response, Request $request)
```
Comment (mickaelandrieu) [inline FrontController/Event/PostResponseEvent.php]: notice ``Request`` object [won't be available with Symfony >= 3](http://symfony.com/blog/new-in-symfony-2-4-the-request-stack), we will inject RequestStack and access Request instance through ``$requestStack->getCurrentRequest()``

File: FrontController/Event/PreRequestEvent.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ *
+ * @category    BackBee
+ * @package     BackBee\FrontController
+ * @copyright   Lp digital system
+ * @author      e.chau <eric.chau@lp-digital.fr>
+ */
+class PreRequestEvent extends Event
+{
+    /**
+     * [__construct description]
```
Comment (mickaelandrieu) [inline FrontController/Event/PreRequestEvent.php]: ?

File: FrontController/Event/PostResponseEvent.php (hunk 1 above)
Comment (eric-chau) [inline FrontController/Event/PostResponseEvent.php]: Ok =)

File: FrontController/Event/PreRequestEvent.php (hunk 2 above)
Comment (eric-chau) [inline FrontController/Event/PreRequestEvent.php]: You're right i will fix it.

**Comment by mickaelandrieu** [comment]
After this PR, would you mind to fix the typo in ``.formatter.yml`` and re-apply formatting and PSR fixer ?

**Pull request merged**